src/server/routes/api.ts

```diff
@@ -0,0 +1,86 @@
+import express from 'express'
+import { Regist } from '../MongooseModels/regist'
+import { logger } from '../logger'
+import { Error } from 'mongoose'
+
+const router = express.Router()
+
+router.get('/', (_, res) => {
+  logger.info('Hey')
+  res.send('Test')
+})
+
+/**
+ * Save a new regist to the DB
+ */
+router.post('/form', (req, res) => {
+  // Example logger to view things. If parameter is a json object must convert to string
```

Comment:
I would like to keep the project clean, so if you dont mind, keep only the necessary comments please